include/flatbuffers/code_generators.h

```diff
@@ -74,9 +74,35 @@ class BaseGenerator {
     return true;
   }
 
-  std::string FullNamespace(const char *separator, const Namespace &ns) {
+  static void NamespacePrefixAsComponents(const std::string &prefix, std::vector<std::string> &namespaces)
+  {
+    std::string::size_type componentStart = 0;
```

Comment:
please use Google C++ style instead of Java style: `component_start`. Same for other variables.